public/js/components/RightSidebar.js

```diff
@@ -192,13 +192,13 @@ const RightSidebar = React.createClass({
 
   getItems() {
     const items = [
-    { header: L10N.getStr("breakpoints.header"),
-      component: Breakpoints,
-      opened: true },
-    { header: L10N.getStr("callStack.header"),
-      component: Frames },
-    { header: L10N.getStr("scopes.header"),
-      component: Scopes }
+      { header: L10N.getStr("breakpoints.header"),
+        component: Breakpoints,
+        opened: true },
+      { header: L10N.getStr("callStack.header"),
+        component: Frames },
+      { header: L10N.getStr("scopes.header"),
+        component: Scopes }
```

Comment:
I prefer the consistency personally, but it's subjective. I can see how this way is okay too.

As a path forward, lets say that the rule is that the keys should be in the same column. If eslint is okay with the first key or last key being on the same line as the curly that's fine...